### PR TITLE
Gisaid script directory

### PR DESCRIPTION
**Pull request by carlacummins**
Move all GISAID conversion tool files into its own directory

### CODE REVIEW

File: scripts/gisaid_to_ena/gisaid_to_ena.py [hunk 1]
```diff
@@ -56,7 +56,7 @@ def parse_args(args):
   Can handle XLS/CSV input, or any format handled by the pandas.read_excel method
   (https://pandas.pydata.org/pandas-docs/stable/reference/api/pandas.read_excel.html)
 
-  Output format can be in Microsoft Excel format (.xlsx) for interactive submissions, 
+  Output format can be in Microsoft Excel format (.xlsx) for interactive submissions,
```
Comment (ahmadazd) [inline scripts/gisaid_to_ena/gisaid_to_ena.py:59]: What do you think of including a note that interactive submission will require converting the excel format into tsv ?

File: scripts/gisaid_to_ena/README.md [hunk 2]
```diff
@@ -0,0 +1,37 @@
+# Converting GISAID metadata to ENA-compatible format
+
+Script to convert GISAID metadata spreadsheets into ENA formatted ones using a mapping of equivalent metadata fields.
+The script can handle CSV, or any format handled by the `pandas.read_excel` method as input ([pandas documentation](https://pandas.pydata.org/pandas-docs/stable/reference/api/pandas.read_excel.html)).
+
```
Comment (ahmadazd) [inline scripts/gisaid_to_ena/README.md:5]: I think it is worth mentioning that in the mapping file, the GISAID headers need to be unique ( no duplicates are allowed).

**Review by ahmadazd — COMMENTED** [review]
Thank you Carla for making this, everything looks great. I left some suggestions as comments

File: scripts/gisaid_to_ena/README.md (hunk 2 above)
Comment (carlacummins) [reply scripts/gisaid_to_ena/README.md:5]: Sure - I will add this, but this is something that we should fix long term. I think you already had an idea of what to do?

File: scripts/gisaid_to_ena/gisaid_to_ena.py (hunk 1 above)
Comment (carlacummins) [reply scripts/gisaid_to_ena/gisaid_to_ena.py:59]: I've added this to the README - thanks

File: scripts/gisaid_to_ena/README.md (hunk 2 above)
Comment (ahmadazd) [reply scripts/gisaid_to_ena/README.md:5]: yes, this should be sorted soon, but for the sake of this user, they should be told in some way, here in the file or by email or something.

**Pull request merged**